src/content/docs/apm/agents/java-agent/instrumentation/java-agent-instrument-kafka-message-queues.mdx

```diff
@@ -164,5 +205,8 @@ To collect distributed traces from Kafka:
       NewRelic.getAgent().getTransaction().acceptDistributedTraceHeaders(TransportType.Kafka, dtHeaders);
     }
     ```
+    <Callout variant="important"> 
+      If you are using Kafka Streams, it is not recommended to accept distributed trace headers because a transaction handles many records.
```

Comment:
Again, suggest that we remove the "not recommended" reference and just point out that the DT will only apply to the first record processed, etc.